ambry-store/src/main/java/com.github.ambry.store/IndexSegment.java

```diff
@@ -343,6 +343,11 @@ short getVersion() {
                 indexFile.getAbsolutePath(), startOffset, keyToFind);
           }
           // binary search on the mapped file
+          if (mmap.isLoaded()) {
+            metrics.indexSegmentInMemoryDuringFindCount.inc();
+          } else {
+            metrics.indexSegmentNotInMemoryDuringFindCount.inc();
```

Comment:
"In-memory" index segment has a different connotation - that of it being a non-memory-mapped managed memory skip list, like the latest index segment. For that reason, I would recommend naming these as `mappedSegmentIsLoadedDuringFindCount` and `mappedSegmentIsNotLoadedDuringFindCount`.

Reply:
good point. I will also add some comments right here.